hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: minor
+
+This release adds :ref:`an internal interface <custom-function-execution>`
```

Comment:
I don't think it's an internal interface - it's in the documentation and everything!